### ISSUE TITLE
NU3018 has different error message from VS and from Docs

### ISSUE THREAD

**Issue by rido-min** [NuGet/docs.microsoft.com-nuget#996]
VS shows:
```
NU3018	The author primary signature found a chain building issue: A certificate chain processed, but terminated in a root certificate which is not trusted by the trust provider.
```
And redirects to this page that has  a different error message:
```
The revocation function was unable to check revocation for the certificate.
```

Only  after reading the details I found that this error can also be caused by Untrusted certs.

IMO we should have consistent error messages across VS and Docs. If the error can be caused by multiple reasons, lead with the most common. Untrusted is going to be more common than "revocation function unavailable".

/c @mishra14 
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fe13f24e-6d6c-f951-6ae5-f79b8cc70ef9
* Version Independent ID: 4f322618-4bf3-fb33-5d7d-4b24469a3c5c
* Content: [NuGet Warning NU3018](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3018)
* Content Source: [docs/reference/errors-and-warnings/NU3018.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/errors-and-warnings/NU3018.md)
* Product: **nuget**
* GitHub Login: @zhili1208
* Microsoft Alias: **lzhi**

**Comment by karann-msft** [comment]
@rido-min we welcome and encourage PRs :)

**Comment by Mikejo5000** [comment]
Looks like this was fixed in #1019
